swagger.yaml

```diff
@@ -401,8 +401,12 @@ paths:
         - in: query
           name: file_name
           type: string
-          enum: [bootstrap.ign, master.ign, metadata.json, worker.ign, kubeadmin-password, kubeconfig, kubeconfig-noingress, install-config.yaml]
+          enum: [bootstrap.ign, master.ign, metadata.json, worker.ign, kubeadmin-password, kubeconfig, kubeconfig-noingress, install-config.yaml, logs]
           required: true
+        - in: query
+          name: host_id
```

Comment:
Sorry I know we agreed about this yesterday but it is doesn't look good.  What do you think about:
`
- in: query
  name: category
  type: string
  enum: [installation, logs, manifests]
  required: true
- in: query
  name: file_name
  type: string
  required: true
`

I think this is more organized, and then if the category is "installation" we will validate that in the backend the file name in the enum, if the category is "logs" then we will validate that it's a valid host ID, if it's manifests we will validate that.

@masayag FYI